gatling-http/src/test/scala/io/gatling/http/HttpSpec.scala

```diff
@@ -40,13 +40,15 @@ import io.netty.channel._
 import io.netty.handler.codec.http._
 import io.netty.handler.codec.http.cookie._
 import org.scalatest.BeforeAndAfter
+import org.scalatestplus.mockito.MockitoSugar
```

Comment:
Please rebase against current heap. You can no use NoopStatsEngine. We're trying to reduce Mockito usage, there's a chance we'll try to drop it some day.